src/components/FormField/FormField.js

```diff
@@ -12,13 +12,13 @@ const FormField = ({
 }) => {
   const [inputValue, setInputValue] = useState(value);
```

Comment:
do you really need state for this component? Shouldn't you use  `value` from props instead?

Reply:
Indeed. Fixed.